indico/modules/events/contributions/templates/display/contribution_display.html

```diff
@@ -239,7 +239,7 @@
         </section>
     {%- endif %}
 
-    {{ render_paper_section(contribution, show_paper=show_paper) }}
+    {{ render_paper_section(contribution, show_paper=contribution.event_new.has_feature('papers')) }}
```

Comment:
I would wrap this macro call in an if checking for the feature instead of having this argument.